ocs_ci/utility/memory.py

```diff
@@ -0,0 +1,197 @@
+# -*- coding: utf8 -*-
+"""
+Module for memory related util functions.
+"""
+
+import os
+import logging
+import tempfile
+from collections import namedtuple
+from itertools import groupby
+import numpy as np
+import pandas as pd
+import psutil
```

Comment:
Instead of this line you can directly import ZombieProcess and NoSuchProcess from line 70 and 72 directly to line number 14
will look like
from psutil import Process, ZombieProcess, NoSuchProcess

Reply:
@shivamdurgbuns done